pybricksdev/cli/__init__.py

```diff
@@ -165,7 +165,9 @@ def add_parser(self, subparsers: argparse._SubParsersAction):
 
     async def run(self, args: argparse.Namespace):
         from ..ble import find_device
-        from ..connections import EV3Connection, PybricksHub, REPLHub
+        from ..connection.ev3dev import EV3Connection
```

Comment:
Small nitpick: module names tend to be plural, e.g. we already have `resources` and `tools`, so I would leave this as `connections`.

Reply:
Done. This is a Java habit where package names are singular 😊

Reply:
Would you advise? Not sure how to compile Python code to avoid compilation issues.

For example, when renaming the `connection` directory name to `connections`, the `cli/*` files were referencing the old module name. `poetry build` was successful but the imports were invalid.

Reply:
I'm not sure I understand the question. There is no compile step for Python. What did you do to trigger the error?

Reply:
IntelliJ underlined the wrong imports by red colour `from ..connection.lego import REPLHub`. Otherwise, I wouldn't even spot this.